testdata/p4_16_samples/loop-3-clause-tricky2.p4

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2024 Andy Fingerhut
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <core.p4>
+#include <v1model.p4>
+
+header ethernet_t {
+    bit<48> dstAddr;
+    bit<48> srcAddr;
+    bit<16> etherType;
+}
+
+struct metadata_t {
+}
+
+struct headers_t {
+    ethernet_t ethernet;
+}
+
+parser parserImpl(packet_in packet,
+                  out headers_t hdr,
+                  inout metadata_t meta,
+                  inout standard_metadata_t stdmeta)
+{
+    state start {
+        packet.extract(hdr.ethernet);
+        transition accept;
+    }
+}
+
+control ingressImpl(inout headers_t hdr,
+                    inout metadata_t meta,
+                    inout standard_metadata_t stdmeta)
+{
+    bit<8> n = hdr.ethernet.srcAddr[15:8];
+    bit<3> i;
+    apply {
+        // Correct sequence of values that i iterates through: 0, 3,
+        // 1, 4
```

Comment:
This is my mistake, but if you wouldn't mind fixing the comment so the list of values i iterates through is correct: 0, 3, 6, 1, 4, that would be appreciated.